src/main/java/ca/warp7/frc2023/lib/SparkMaxUtil.java

```diff
@@ -0,0 +1,83 @@
+package ca.warp7.frc2023.lib;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.MotorFeedbackSensor;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.wpilibj.Encoder;
+
+public class SparkMaxUtil {
```

Comment:
Reiterating... Please avoid creating a lib file for something like this. You can just configure your motors in a method in your subsystem.